src/thunderbots/shared/constants.h

```diff
@@ -51,3 +51,6 @@ const double MILLISECONDS_PER_MICROSECOND = 1.0 / 1000.0;
 
 // Converts dribbler RPM to a smaller number firmware uses
 const double DRIBBLER_RPM_TO_RADIO_CONVERSION_FACTOR = 1.0 / 300.0;
+
+const double POSSESSION_BUFFER_TIME_IN_SECONDS              = 3.0;
```

Comment:
nit? Empirically I feel like this should be ~1.5 seconds. I don't think the ball will be "loose" for that long with the other team still being considered to have possession (or us for that matter. The ball travel time for our passes is < 1.5 seconds).

@henrymliu @EvanMorcom what do you think?

Reply:
Would agree 3 is a bit too much, anything between 1 and 2 seconds seems reasonable to me.

Reply:
Changed to 1.5